task/clair-scan/0.2/clair-scan.yaml

```diff
@@ -92,22 +92,32 @@ spec:
           value: $(params.image-url)
         - name: IMAGE_DIGEST
           value: $(params.image-digest)
+      workingDir: /tekton/home
       script: |
         #!/usr/bin/env bash
 
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
         imagewithouttag=$(echo -n $IMAGE_URL | sed "s/\(.*\):.*/\1/")
         images_processed_template='{"image": {"pullspec": "'"$IMAGE_URL"'", "digests": [%s]}}'
         digests_processed=()
 
-        for sha_file in /tekton/home/image-manifest-*.sha; do
+        clair_report() {
+          { clair-action report --image-ref="$1" --db-path=/tmp/matcher.db --format=quay | tee "clair-result-$arch.json"; } && \
+          { clair-action report --image-ref="$1" --db-path=/tmp/matcher.db --format=clair > "clair-report-$arch.json"; }
```

Comment:
What's the performance impact of running `clair-action` twice? Not sure how long each command takes to run.

Reply:
In the past 30 days most seem to end within < 30 sec (65%), I would guess that this correlates to the image size. I don't see any caching in claircore or clair-action, so it will be at least double that

Reply:
Ack. That's probably ok.